DOCSP-33172 added meta description for top 250 pages in docs-mongodb-shell (#307)
mongodb/docs-mongodb-shell: source/crud/update.txt

   :values: reference

.. meta:: 

>    :description: MongoDB provides methods to update a single document, update multiple documents, and replace a document. 

Reviewer: ```suggestion
   :description: Use the MongoDB Shell to update a single document, update multiple documents, or replace a document. 
```
